examples/online-store/flows/LoadOrders.flow.json

```
@@ -1,11 +1,22 @@
 {
   "name" : "LoadOrders",
   "description" : "flow description",
-  "identifier" : "",
+  "batchSize" : 100,
+  "threadCount" : 4,
+  "stopOnError" : false,
+  "options" : null,
+  "version" : 0,
   "steps" : {
-    "1": {
-      "stepDefinitionName": "IngestOrders",
-      "stepDefinitionType": "CUSTOM"
+    "1" : {
+      "name" : "IngestOrders",
+      "description" : "null",
+      "options" : { },
+      "customHook" : null,
+      "retryLimit" : 0,
+      "batchSize" : 0,
+      "threadCount" : 0,
```

Should be null or matching the flow
```suggestion
      "threadCount" : null,
```